lib/aruba/processes/spawn_process.rb

```diff
@@ -267,7 +270,7 @@ def read_temporary_output_file(file)
         data = file.read
         file.close
 
-        data
+        data.force_encoding('UTF-8')
```

Comment:
@enkessler There seem to be some encoding issues with childprocess. Can you help us with that?